src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -43,11 +43,11 @@ static DistributeObjectOps Aggregate_AlterOwner = {
 };
 static DistributeObjectOps Aggregate_Define = {
 	.deparse = NULL,
```

Comment:
fascinating, did we skip the deparse of the statement?
I guess this can be done because we only handle the postprocess stage, where we can reconstruct the sql from the catalog

Reply:
We have `GetAggregateDDLCommand(const RegProcedure funcOid, bool useCreateOrReplace)`